pkgs/applications/misc/rsclock/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname= "rsClock";
+  version= "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "valebes";
+    repo = pname;
+    rev ="49335356cd6d0544491cd97ede52e7e0ec027da2";
```

Comment:
Do you mind either creating a github release of this software or move it to unstable as per https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Ok